packages/api-client/src/user/UserAPI.ts

```diff
@@ -51,11 +51,19 @@ type PrekeysResponse = {
   qualified_user_client_prekeys: QualifiedUserPreKeyBundleMap;
   failed_to_list?: QualifiedId[];
 };
-
 function isPrekeysResponse(object: any): object is PrekeysResponse {
   return object.qualified_user_client_prekeys && object.failed_to_list;
 }
 
+type UsersReponse = {
+  found?: User[];
+  failed?: QualifiedId[];
+  not_found?: QualifiedId[];
+};
+function isUsersResponse(object: any): object is UsersReponse {
+  return object.found || object.failed || object.not_found;
```

Comment:
why the param named object? can be give it a meaningful name like?

Reply:
That's because it's a typeguard predicate. So we basically give it something, and we don't know what that thing is, and in return it tells us if that thing is a `UserResponse`